scripts/build/sign_packages.sh

```diff
@@ -1,12 +1,19 @@
 #!/bin/bash
 
-git clone git@github.com:torkelo/private.git ~/private-repo
+set -e
 
-gpg --allow-secret-key-import --import ~/private-repo/signing/private.key
+if [ -z "$GPG_KEY_PASSWORD" ]; then
+    echo "GPG_KEY_PASSWORD has to be set"
+    exit 1
+fi
+
+_files=$*
+
+gpg --allow-secret-key-import --import /private.key
```

Comment:
This makes the script problematic to run multiple times. Not sure what the best solution would be, either move this part to separate script or make gpg to not exit the script (remove `set -e`?)

Reply:
I think the import should be done outside of the script. Alternatively, we could have a script that import the key and then calls the actual signing script. But that would at least make it possible to run the signing script multiple times without running into problems.